CONTRIBUTING.md

```
@@ -165,14 +183,90 @@ refactor: simplify Twitch notification logic
+- changes
+```
+
+4. Commit the generated changeset file:
+```bash
+git add .changeset/*.md
+git commit -m "feat: add forum channel support"
+```
+
+### Changeset Flow
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)